decidim-consultations/app/views/decidim/consultations/admin/consultations/results.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="card" id="consultations">
+  <div class="card-divider">
+    <h2 class="card-title">
+      <%= t "decidim.admin.titles.results" %>
+      <span class="label button--title"><%= t "decidim.admin.consultations.results.total_votes", count: current_consultation.total_votes %></span>
+    </h2>
+  </div>
+  <div class="card-section">
+
+    <% current_consultation.questions.published.each do |question| %>
+      <% unless question.external_voting %>
+        <table class="table-list">
+          <thead>
+            <tr>
+              <th><%= decidim_sanitize translated_attribute question.title %></th>
+              <th class="table-list__actions">
+                <%= t "decidim.admin.consultations.results.total_votes", count: question.total_votes %>
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            <% if current_consultation.finished? && question.sorted_results.any? %>
```

Comment:
What about moving this logic to the question model? Something like:

```ruby
def public_results?
  consultation.finished? && sorted_results.any?
end
```

Sorry for not adding this comment before, I don't want to seem I keep finding things to change 😅

Reply:
don't worry, you're right. But maybe the method should not be called public_results as public means when are published. Maybe `publishable_results?`